src/screens/ChooseActivity.tsx

```diff
@@ -32,7 +31,7 @@ const PatientScreen: React.FC<ScreenProps> = ({ navigation, route }) => {
 
   const handleListPress = (index: number) => {
     const activ = activities[index];
-    Alert.alert(`(${index}) ${activ.name} (${activ.shortName}) selecionado!`);
+    if (activ) navigation.navigate("CarouselScreen");
```

Comment:
Acho que aqui devemos usar um `navigation.reset(...)`, pra evitar deixar o usuário voltar pra essa tela quando já estiver na tela de execução:

```suggestion
    if (activ) navigation.reset({ index: 0, routes: [{ name: "CarouselScreen" }] });
```

Reply:
Mudança feita.